capnp-rpc/Cargo.toml

```diff
@@ -29,3 +29,6 @@ capnp = {version = "0.9.0", path = "../capnp",  features = ["rpc"]}
 # prevent 'cargo test' from trying to build the examples
 name = "dummy"
 path = "examples/dummy.rs"
+
+[features]
```

Comment:
Do we gain anything by adding this feature here? It seems to me that anyone wanting to use it could more simply enable it directly on the capnp crate.

Reply:
No, you are right. I've pushed a fix to this.